kernel/src/error.rs

```diff
@@ -171,6 +171,9 @@ pub enum Error {
     /// The file already exists at the path, prohibiting a non-overwrite write
     #[error("File already exists: {0}")]
     FileAlreadyExists(String),
+
+    #[error("Change data feed is disabled in range: {0} to {}", _1.map_or("end".into(), |ver| ver.to_string()))]
```

Comment:
This error message has some confusing syntax, but I think we should clarify here. The error is not because it's disabled in the range (or the entire range perhaps), there is nothing wrong with doing that but that CDF doesn't allow you to disable it anywhere in the range and still work.